bin/edit.js

```diff
@@ -26,6 +35,10 @@ function handleEdit(args) {
     var wtName  = args.name ? args.name + '/' : '';
     var url     = profile.url + '/edit/' + profile.container + '#/' + wtName + profile.token;
 
+    if (args.previousVersion) {
+        url = profile.url + '/prev/edit/' + profile.container + '#/' + wtName + profile.token;
```

Comment:
We can revert the major package to an specific patch easily and quickly so I think by having `major` support on the CLI will work :).